spec/src/main/asciidoc/chapters/language/syntax.adoc

```diff
@@ -12,11 +13,89 @@ The following grammar is the grammar for the core language.
 ----
 
+[[full-grammar]]
 === Full language grammar
 
 This much longer grammar is the grammar for the full language.
 
 [source, antlrv4]
 ----
 include::../../../../antlr/JQL-full.g4[]
-----
\ No newline at end of file
+----
+
+=== Language Levels and Feature Summary
+
+The Core and Full grammars together define two levels of Jakarta Query: *Core* and *Extended*.
+The Core level represents the minimal, portable subset, while the Extended level is a superset that introduces SQL-oriented features such as joins, grouping, and subqueries.
+This subsection summarizes the differences between them, based on the syntax defined in the <<Core language grammar>> and the <<Full language grammar>> sections.
+
+[NOTE]
+====
+In the following tables:
+* Yes = Feature available at this level
+* No  = Feature not available at this level
+====
+
+==== Clauses and Constructs
+
+[cols="3,1,1",options="header"]
+|===
+| Feature / Construct                               | Core | Extended
+| `FROM` clause                                     | Yes | Yes
+| `WHERE` clause                                    | Yes | Yes
+| `ORDER BY` clause                                 | Yes | Yes
+| `SELECT` paths/id/count                           | Yes | Yes
+| Update (`UPDATE ... SET`)                         | Yes | Yes
+| Delete (`DELETE FROM`)                            | Yes | Yes
+| Joins (inner/left/fetch)                          | No  | Yes
+| Grouping (`GROUP BY`)                             | No  | Yes
+| Having (`HAVING`)                                 | No  | Yes
+| Aggregates (`SUM`, `AVG`)                         | No  | Yes
+| Distinct in `SELECT`                              | No  | Yes
+| Constructor expressions                           | No  | Yes
+| Subqueries (`EXISTS`, `IN`)                       | No  | Yes
+| Set operations (`UNION`, `INTERSECT`, `EXCEPT`)   | No  | Yes
+|===
+
+==== Special Expressions and Functions
+
+===== Datetime Expressions
+
+[cols="3,1,1",options="header"]
+|===
+| Expression                                          | Core | Extended
+| `CURRENT_DATE`, `CURRENT_TIME`, `CURRENT_TIMESTAMP` | Yes | Yes
+| `EXTRACT(field FROM x)`                             | No  | Yes
+|===
+
+===== Boolean Expressions
+
+[cols="3,1,1",options="header"]
+|===
+| Expression                 | Core | Extended
+| Literals `TRUE`, `FALSE`   | Yes | Yes
+| `EXISTS`                   | No  | Yes
+|===
+
+===== Numeric Expressions
+
+[cols="3,1,1",options="header"]
+|===
+| Expression                   | Core | Extended
+| Basic operators (+, -, *, /) | Yes | Yes
+| `ABS`                        | Yes | Yes
+| `MOD`                        | No  | Yes
+| `ROUND`                      | No  | Yes
+|===
+
+===== String Expressions
+
+[cols="3,1,1",options="header"]
+|===
+| Expression                             | Core | Extended
+| &#124;&#124; concatenation             | Yes | Yes
+| `LENGTH`, `LOWER`, `UPPER`             | Yes | Yes
```

Comment:
These should probably be split up as well.  LOWER and UPPER make sense together.  LENGTH does something very different, and even returns a different type
```suggestion
| `LENGTH`                               | Yes | Yes
| `LOWER`, `UPPER`                       | Yes | Yes
```